parsl/tests/test_radio/test_basic.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import os
+import parsl
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+@parsl.python_app
+def this_app():
+    # this delay needs to be several times the resource monitoring
+    # period configured in the test configuration, so that some
+    # messages are actually sent - there is no guarantee that any
+    # (non-first) resource message will be sent at all for a short app.
+    import time
+    time.sleep(3)
+
+    return 5
+
+@pytest.mark.local
+def test_energy_collection():
```

Comment:
this is a weird test name, I think from some other work?

Reply:
Yeah you are right. I forgot to change this name.